cmd/dashboard.go

```diff
@@ -50,10 +50,9 @@ var DashboardCmd = &cobra.Command{
 			os.Exit(0)
 		}
 
-		if port < 0 {
-			localPort = defaultLocalPort
-		} else {
-			localPort = port
+		if dashboardLocalPort <= 0 {
+			print.FailureStatusEvent(os.Stdout, "Invalid port: %v", dashboardLocalPort)
```

Comment:
We need to document this change of behavior

Reply:
Where do we document these things?